### ISSUE TITLE
los números telefónicos

### ISSUE THREAD

**Issue by CrimsonGlory** [CrimsonGlory/posaderos#105]
Estaba leyendo sobre como se deberían almacenan los teléfonos.
Para nuestro uso lo actual está bien, ya que es algo que se va a usar poco (o nada) y son números de Buenos Aires. (11)
Pero por si algún día se le da un uso más amplio deberían hacerse los siguientes cambios:
[ ] Crear una clase Phone. polimórfica many-to-many. Ya que tanto Person como User pueden tener muchos phones.
[ ] en la base de datos, la tabla phones va a tener las siguientes columnas: cc=country code (int), ndc=national destination code (varchar(14)), sn=suscriber number(varchar(14)).
ndc y sn serían varchar porque podrían empezar con cero. En Argentina el ndc nunca empieza con cero (porque cero es el prefijo de larga distancia), pero no se en el resto del mundo.

Links relevantes
https://en.wikipedia.org/wiki/List_of_country_calling_codes
https://en.wikipedia.org/wiki/E.164
https://en.wikipedia.org/wiki/E.123
https://github.com/floere/phony
https://es.wikipedia.org/wiki/N%C3%BAmeros_telef%C3%B3nicos_en_Argentina
http://www.cnc.gob.ar/numeracion-geografica_p137

**Comment by aguspuentes** [comment]
Claro, eso sería la posta para guardar bien los teléfonos. La realidad es que en el caso de los asistidos calculo que casi ninguno tiene teléfono y en el caso de los usuarios sólo es informativo, ya que en general se comunican vía correo electrónico entre ellos.
Igual me parece bien dejar esta issue acá, en caso que en el futuro se decida optimizar la aplicación.
